presto-hive/src/test/java/io/prestosql/plugin/hive/AbstractTestHive.java

```diff
@@ -2624,15 +2624,15 @@ public void testHideDeltaLakeTables()
                         .doesNotContain(tableName);
 
                 // list all columns
-                assertThat(metadata.listTableColumns(session, new SchemaTablePrefix()).keySet())
+                assertThat(metadata.listTableColumnsWithRedirection(session, new SchemaTablePrefix()).keySet())
                         .doesNotContain(tableName);
 
                 // list all columns in a schema
-                assertThat(metadata.listTableColumns(session, new SchemaTablePrefix(tableName.getSchemaName())).keySet())
+                assertThat(metadata.listTableColumnsWithRedirection(session, new SchemaTablePrefix(tableName.getSchemaName())).keySet())
```

Comment:
Why the change here?
Is `listTableColumns` deprecated and should not be called?

Or, we should test both: `listTableColumns` and `listTableColumnsWithRedirection` here?

(this comment applies to other test places as well)

Reply:
`listTableColumns` is removed from `HiveMetadata`